Assignment/solution_student/css/style.css

```diff
@@ -0,0 +1,138 @@
+/*GLOBAL STLES*/
+
+html {
+	padding: 1%;
+	border-bottom: 15px solid #005C89;
+	border-top: 15px solid #005C89;
+}
+
+nav {
+	text-align: center;
+	font-family: Open Sans, sans-serif;
+}
+
+
+body {
+	background-color: #f6f6f6;
+
+}
+
+img {
+	width: 90%;
+	border: 2px solid #005C89;
+	padding: 3px;
+}
+
+h1 {
+	font-size: 60px;
+	font-family: Lora, Serif;
+	color: #272A32;
+}
+
+h2 {
+	font-family: Lora, Serif;
+	font-size: 30px;
+	color: #272A32;
+}
+
+h3 {
+	font-family: Lora, Serif;
+	font-size: 24px;
+	color: #272A32;
+}
+
+h4 {
+	font-family: Open Sans, sans-serif;
+	font-size: 21px;
+}
+
+p {
+	line-height: 1.5em;
+	font-size: 21px;
+}
+
+ul li span {
+	color: #272A32;
+}
+
+ul {
+	list-style: disc;
+	color: #FF0028;
+}
+
+hr {
+	color: #FF0028;
+}
+
+a {
+	text-decoration: none;
+	color: #005C89;
+}
+
+a:hover {
+	color: #FF0028;
+	text-decoration: underline;
+}
+
+li {
+	font-family: Open Sans, sans-serif;
+	padding-bottom: 20px;
+	font-size: 21px;
+}
+
+li p {
+	font-family: Lora, serif;
+}
+
+.all {
+	width: 70%;
+	margin: auto;
+
+}
+
+.footer {
+	text-align: center;
+	font-family: Open Sans, sans-serif;
+	font-size: 16px;
+	clear: both;
+}
+
+
+
+
+
+/*INDEX.HTML STYLES*/
+
+
+
+
+
+
+/*PORTFOLIO.HTML STYLES*/
+
+.info {
+	float: left;
+	max-width: 60%;
+}
+
+.portrait {
+	border-radius: 100%;
+	max-width: 30%;
+	float: right;
+	margin: 20px;
+}
+
+
+/* float/percentage discrepancy quick fix!*/
+
+#fake-space {
```

Comment:
Great resourcefulness on this one! Floats are very tricky, and we'll be going over this on Thursday, but the most widely accepted hack (yes, unfortunately when using floats you are actually forced to use a hack) is call the clearfix hack, which is this: apply the `.clearfix` class to the element that contains the floated elements (which in your case is `<div class="all">`), which will add a full-with block-level pseudo-element `:before` and `:after` the element, forcing it to occupy the amount of space it should, and thus not breaking the layout like you discovered. The clearfix class looks like this:
```
.clearfix:before,
.clearfix:after {
    content: " "; /* 1 */
    display: table; /* 2 */
}
.clearfix:after {
    clear: both;
}
```
You can read more about it here: http://nicolasgallagher.com/micro-clearfix-hack/